index.html

```diff
@@ -0,0 +1,1342 @@
+            </div>
+
+        </div>
+
+        <div class="newsletter bg-light d-flex flex-column w-100 w-md-auto d-md-none">
+            <h5>NEWSLETTER</h5>
+            <div class=" newsletter-body d-flex h-100 flex-column">
+                <div class="newsletter-img">
+                    <img src="./assets/sub/starting5-homepage.png" alt="">
+                </div>
+                <div class="star-your-day text-center">
+                    <h5 class="fw-bold">THE STARTING 5</h5>
+                    <p>See why nearly a quarter of a million subscribers begin their day with the Starting 5.</p>
+                    <div class="sign-up-div d-flex">
+                        <input type="text" class="form-control flex-grow-1 w-25" placeholder="Email Address"
+                            aria-label="Email">
+                        <button class="btn btn-danger flex-grow-1">FREE SIGN UP <i class="fa fa-chevron-right"
+                                aria-hidden="true"></i></button>
+                    </div>
+                </div>
+            </div>
+
+        </div>
+        <div class="last-card-sub d-lg-flex">
+            <div class="subscribe bg-light d-none d-md-flex p-4">
+                <div class="img-b position-relative">
+                    <img class="position-absolute pic-n-1" src="./assets/magazine/cover-january-29-february-05-2021.jpg"
+                        alt="">
+                    <img class="position-absolute pic-n-2" src="./assets/magazine/cover-january-29-february-05-2021.jpg"
+                        alt="">
+                </div>
+                <div class="info-b p-3">
+                    <h3>In-depth Stories You Don't See Elsewhere About Topics You Want to Read!</h3>
+                    <p>All Access + Weekly Delivery. <br> Choose A Membership That's Perfect for You!</p>
+                    <button class="btn btn-danger rounded">SUBSCRIBE</button>
+                </div>
+            </div>
+
+            <div class="newsletter bg-light d-none d-md-flex p-3">
+                <div class="img-n col-lg-6  overflow-hidden">
+                    <img src="./assets/sub/starting5-homepage.png" alt="">
+                </div>
+                <div class="info-n d-flex col-lg-6  text-center justify-content-between flex-column p-4">
+                    <h5 class="fw-bold">THE STARTING 5</h5>
+                    <p>See why nearly a quarter of a million subscribers begin their day with the Starting 5.</p>
+                    <div class="sign-up-div d-flex flex-lg-column">
+                        <input type="text" class="form-control flex-grow-1 w-25 w-lg-auto" placeholder="Email Address"
+                            aria-label="Email">
+                        <button class="btn btn-danger flex-grow-1">FREE SIGN UP <i class="fa fa-chevron-right"
+                                aria-hidden="true"></i></button>
+                    </div>
+                </div>
+
+            </div>
+        </div>
+    </div>
```

Comment:
- [ ] Fix the styling and text of the 'Subscribe' and 'Newsletter' sections
- [ ] Add the red titles in the top of each section.

**On desktop:**
![subscribe_and_newsletter_desktop](https://user-images.githubusercontent.com/16193212/106056447-6db3e380-60cd-11eb-8fa1-d1e53314e621.png)
**On Tablet:**
![subscribe_and_newsletter_tablet](https://user-images.githubusercontent.com/16193212/106057379-aa340f00-60ce-11eb-9e3b-1a77f83adcef.png)

**On Mobile:**
![subscribe_and_newsletter_mobile](https://user-images.githubusercontent.com/16193212/106057400-b029f000-60ce-11eb-90e8-a77e57fab946.png)